include/ozo/connection.h

```diff
@@ -63,6 +134,32 @@ struct get_connection_socket_impl {
     }
 };
 
+/**
+ * @ingroup Connection_Concept
+ * @brief Get the connection socket object
+ *
+ * Connection socket object getter. Connection must provide socket for IO. In general it must return
+ * reference to an instance of [boost::asio::posix::stream_descriptor](https://www.boost.org/doc/libs/1_67_0/doc/html/boost_asio/reference/posix__stream_descriptor.html).
```

Comment:
We support boost 1.66 in CI, better use links to this version.

Reply:
**Done**